src/main/scala-2.12/org/ergoplatform/appkit/ergotool/dex/CreateBuyOrderCmd.scala

```diff
@@ -97,19 +95,26 @@ object CreateBuyOrderCmd extends CmdDescriptor(
     val args = ctx.cmdArgs
     val storageFile = new File(if (args.length > 1) args(1) else error("Wallet storage file path is not specified"))
     if (!storageFile.exists()) error(s"Specified wallet file is not found: $storageFile")
-    val buyer = Address.create(if (args.length > 2) args(2) else error("buyer address is not specified"))
-    val ergAmount = if (args.length > 3) args(3).toLong else error("ergAmount is not specified")
-    val tokenId = if(args.length > 4) args(4) else error("tokenId is not specified")
-    val tokenAmount = if(args.length > 5) args(5).toLong else error("tokenAmount is not specified")
+    val ergAmount = if (args.length > 2) args(2).toLong else error("ergAmount is not specified")
```

Comment:
why not use parameters descriptors here?

Reply:
Because it'll all be gone after merging #19.